README.md

```diff
@@ -51,6 +51,21 @@ ptyProcess.resize(100, 40);
 ptyProcess.write('ls\r');
 ```
 
+## Prebuilt Binaries
```

Comment:
I think the Building section should be above this one

Reply:
Ok, will change.